docs/userguides/cache.md

```diff
@@ -0,0 +1,77 @@
+# Cache
+
+Use the cache plugin to store provider data in an sqlite database.
+
+```bash
+ape cache init --network <your-choice-of-network>
+```
+
+You will need to set-up your network connection.
+
+For example, if you want to work with an Infura provider, you need to install the Infura plugin.
+
+```bash
+ape plugins install infura
+```
+
+Then, to connect to the mainnet:
+
```

Comment:
I wouldn't mention any of this, if you want to link to that we have a user guide on using a provider.

Also, due to the changes we made we don't require a network connection to init

Reply:
[2639da5](https://github.com/ApeWorX/ape/pull/680/commits/2639da5da814458917bc963229c0d89ad367d068)